subset/network/network_tests.py

```diff
@@ -43,23 +43,23 @@ def shell_command_with_result(command, wait_time, terminate_flag):
     if len(text) > 0:
         return text
 
-def validate_test():
+def add_packet_info_to_report():
     max = 0
     if packets_received > packets_in_report :
         max = packets_in_report
     else:
         max = packets_received
     for i in range(0, max):
         write_report(packet_request_list[i] + '\n')
-    write_report('packets_sent=' + str(packets_received)  + '\n')
-    write_report("RESULT pass %s\n" % test_request)
+    write_report("packets_sent=%s\n") % str(packets_received)
```

Comment:
syntactically incorrect. Also the str() is redundant (automatically done for %s argument, I believe)